app/src/main/res/values-ja/strings.xml

```diff
@@ -1,64 +1,104 @@
-<?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <string name="app_name">食品の賞味期限</string>
-    <string name="app_name_short">@string/app_name</string>
+    <string name="app_name">Food Expiration Dates</string>
+    <string name="app_name_short">Food Expirations</string>
```

Comment:
Is it good not to translate the name of the app? 
The app also has this name so that people can find it on Google Play: if an English-speaking person wants to find an app for keeping track of expiration dates, he/she will probably type in "expirations for food," "expiration tracker," or something similar. What would a Japanese person type into the search bar when looking for this kind of app?

Reply:
Even if the app supports Japanese, there are many cases where the app name is not translated into Japanese. (For example, Instagram, Spotify, YouTube, etc. remain the same as English even on devices set to Japanese.)
Of course, there are some foreign apps whose names are also translated into Japanese, but I chose the former.
Please feel free to let us know if you have any questions.

Reply:
Even if the app supports Japanese, there are many cases where the app name is not translated into Japanese. (For example, Instagram, Spotify, YouTube, etc. remain the same as English even on devices set to Japanese.)
Of course, there are some foreign apps whose names are also translated into Japanese, but I chose the former.
Please feel free to let us know if you have any questions.

Reply:
Understood, thanks for your explaination 🙂